### PR TITLE
VISTA Ballot Areas

### PR DESCRIPTION
**Pull request by miriamseely**
Current draft for the metadata, with some of my comments and questions mixed in. Let me know what you think!

### CODE REVIEW

**Review by stdavis — COMMENTED** [review]
This looks great to me!

I didn't realize that not every county used this dataset. It seems to have data for every county. Do you have more info about this?

File: metadata/political/vista_ballot_areas.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+# Title
+
+Utah VISTA Ballot Areas
+
+<!--- VISTA or Vista? I've seen it both ways in the metadata and on our blog. Also, I noticed the information on this dataset mostly refers to the polygons as "voting precincts" rather than ballot areas, should we change the title accordingly? -->
```
Comment (steveoh) [inline metadata/political/vista_ballot_areas.md:5]: From Mark Mitchell, 
> VISTA is the name of our election management system.  It stands for Voter Information and State Tracking Application, an acronym, so it should all be capitalized.

File: metadata/political/vista_ballot_areas.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+fb485021-1a93-43b3-bd3b-13c68d7e5b3c
+
+## Brief Summary
+
+Official voting precincts used by Utah's VISTA election framework.
+
+## Summary
+
+Statewide polygon layer that depicts voting precinct and subprecinct boundaries. This dataset serves as the baseline for VISTA, Utah's election registration database, as well as many other voter information websites.
```
Comment (steveoh) [inline metadata/political/vista_ballot_areas.md:17]: From Mark Mitchell
```suggestion
Statewide polygon layer that depicts voting precinct and subprecinct boundaries. This dataset serves as the baseline for VISTA, Utah's voter registration database, as well as many other voter information websites.
```

File: metadata/political/vista_ballot_areas.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+## Description
+
+### What is the dataset?
+
+Voting precincts are used to determine a voter's ballot composition and polling location. A number of factors contribute to this, including the boundaries of all political districts covering the voter's address (U.S. Congress, State House and Senate, State School Board, municipal districts, etc.) and the legislatively-mandated maximum limit of voters per precinct.
```
Comment (steveoh) [inline metadata/political/vista_ballot_areas.md:23]: From Mark Mitchell
```suggestion
Voting precincts are used to determine a voter's ballot composition and polling location. A number of factors contribute to this, including the boundaries of all political districts covering the voter's address (U.S. Congress, State House and Senate, State School Board, local county and municipal districts, etc.) and the legislatively-mandated maximum limit of voters per precinct.
```

File: metadata/political/vista_ballot_areas.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+### What is the purpose of the dataset?
+
+This dataset is actively used in more than 20 counties in Utah. These are counties that have adopted a GIS-based approach to manage precinct-to-residence assignment within VISTA, the statewide voter registration database. This dataset also drives many of the State's voter information pages and helps citizens know [how and where to vote](https://votesearch.utah.gov/voter-search/search/search-by-address/how-and-where-can-i-vote).
```
Comment (steveoh) [inline metadata/political/vista_ballot_areas.md:27]: From Mark Mitchell
```suggestion
This dataset is actively used in all 29 counties in Utah. The counties have adopted a GIS-based approach to manage precinct-to-residence assignment within VISTA, the statewide voter registration database. This dataset also drives many of the State's voter information pages and helps citizens know [how and where to vote](https://votesearch.utah.gov/voter-search/search/search-by-address/how-and-where-can-i-vote).
```

File: metadata/political/vista_ballot_areas.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+
+This dataset contains polygons representing more than 3000 voting precincts and subprecincts in Utah. Each feature contains attributes such as the date the precinct boundary took effect (effectivedate) and whether the polygon contains a subprecinct boundary (indicated, where applicable, in the subprecinct field).
+
+<!--- This next paragraph is from the original metadata. I haven't seen other data pages explain GIS operations for a dataset in this way, is this something we generally like to include? -->
+
+Precincts can be formed (and subprecinct divisions eliminated) by performing a GIS ‘dissolve’ operation on the precinctid field.
+
+### How was the dataset created?
+
+UGRC aggregates voter precinct data submitted by County Clerk offices. This dataset is updated periodically as municipal and county boundaries change and as voter count grows beyond the maximum number of voters allowed per precinct (currently 1250 voters).
```
Comment (steveoh) [inline metadata/political/vista_ballot_areas.md:41]: From Mark Mitchell
```suggestion
UGRC aggregates voter precinct data submitted by County Clerk and GIS offices. This dataset is updated periodically as municipal and county boundaries change and as voter count grows beyond the maximum number of voters allowed per precinct (currently 1250 voters).
```

**Review by steveoh — COMMENTED** [review]
I reached out to mark for comment and this is what he put in a word doc.

**Comment by steveoh** [comment]
> I didn't realize that not every county used this dataset

I think Marks comments resolve that concern.

File: metadata/political/vista_ballot_areas.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+
+This dataset is actively used in all 29 counties in Utah. The counties have adopted a GIS-based approach to manage precinct-to-residence assignment within VISTA, the statewide voter registration database. This dataset also drives many of the State's voter information pages and helps citizens know [how and where to vote](https://votesearch.utah.gov/voter-search/search/search-by-address/how-and-where-can-i-vote).
+
+### What does the dataset represent?
+
+This dataset contains polygons representing more than 3000 voting precincts and subprecincts in Utah. Each feature contains attributes such as the date the precinct boundary took effect (effectivedate) and whether the polygon contains a subprecinct boundary (indicated, where applicable, in the subprecinct field).
+
+<!--- This next paragraph is from the original metadata. I haven't seen other data pages explain GIS operations for a dataset in this way, is this something we generally like to include? -->
+
+Precincts can be formed (and subprecinct divisions eliminated) by performing a GIS ‘dissolve’ operation on the precinctid field.
```
Comment (jacobdadams) [inline metadata/political/vista_ballot_areas.md:33]:
```suggestion
Precincts can be formed by eliminating subprecinct divisions by dissolving the features on the precinctid field.
```
I think this is important, as it talks about how you can derive precincts from the dataset. Maybe we just make it a little less "tutorial-ly"?

File: metadata/political/vista_ballot_areas.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+
+### How was the dataset created?
+
+UGRC aggregates voter precinct data submitted by County Clerk and GIS offices. This dataset is updated periodically as municipal and county boundaries change and as voter count grows beyond the maximum number of voters allowed per precinct (currently 1250 voters).
```
Comment (jacobdadams) [inline metadata/political/vista_ballot_areas.md:37]:
```suggestion
UGRC aggregates voter precinct data submitted by County Clerk and GIS offices. They send periodic updates as municipal and county boundaries change and as voter count grows beyond the maximum number of voters allowed per precinct (currently 1250 voters).
```

**Pull request merged**